<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Вираз множини:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Множина-операнд:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Множина B:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Множина D:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="set_b">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>{1, 2, 3, 4}</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="next_step">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Наступний крок</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Множина А:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="set_d">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>{1, 2, 3, 4}</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="expression">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>A U B</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="set_a">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>{1, 2, 3, 4}</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="result">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>{1, 2, 3, 4}</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="set_c">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>{1, 2, 3, 4}</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-family: Gill Sans, sans-serif;
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Множина C:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="last_step">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Попередній крок</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="save_to_file">
        <property name="text">
         <string>Зберегти до файлу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="information">
   <property name="text">
    <string>Про студента</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Зберегти до файлу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
